django/thunderstore/api/cyberstorm/views/team.py

```diff
@@ -0,0 +1,67 @@
+from django.db.models import Q, QuerySet
+from rest_framework.exceptions import NotFound, PermissionDenied
+from rest_framework.filters import OrderingFilter
+from rest_framework.generics import ListAPIView, RetrieveAPIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.request import Request
+
+from thunderstore.account.models.service_account import ServiceAccount
+from thunderstore.api.cyberstorm.serializers import (
+    CyberstormServiceAccountSerializer,
+    CyberstormTeamMemberSerializer,
+    CyberstormTeamSerializer,
+)
+from thunderstore.api.utils import CyberstormAutoSchemaMixin
+from thunderstore.repository.models.team import Team, TeamMember
+
+
+class TeamDetailAPIView(CyberstormAutoSchemaMixin, RetrieveAPIView):
+    serializer_class = CyberstormTeamSerializer
+    queryset = Team.objects.exclude(is_active=False)
+    lookup_field = "name__iexact"
+    lookup_url_kwarg = "team_id"
+
+
+class TeamRestrictedAPIView(ListAPIView):
+    """
+    Ensure the user is a member of the Team.
+    """
+
+    permission_classes = [IsAuthenticated]
+
+    def check_permissions(self, request: Request) -> None:
+        super().check_permissions(request)
+
+        try:
+            team = Team.objects.exclude(is_active=False).get(
+                name__iexact=self.kwargs["team_id"],
+            )
+        except Team.DoesNotExist:
+            raise NotFound()
+
+        if not team.can_user_access(request.user):
+            raise PermissionDenied()
+
+
+class TeamMembersAPIView(CyberstormAutoSchemaMixin, TeamRestrictedAPIView):
+    serializer_class = CyberstormTeamMemberSerializer
+    filter_backends = [OrderingFilter]
```

Comment:
Only now realized this isn't using the in-project `StrictOrderingFilter` that addresses a few minor issues with the DRF-provided `OrderingFilter` (check comment in the implementation for details).

Noting for future reference, not a blocker for this PR